test/error.test.js

```diff
@@ -43,7 +43,13 @@ describe('Handling Errors ', function() {
         new mapnik_backend({pathname:'./test/data/invalid_font_face.xml'}, function(err, source) {
             try {
               assert.ok(!err, err);
-              done();
+              if (source) {
+                  source.close(function(err) {
```

Comment:
Should the `close` callback call the `done` function?

Reply:
I copy & paste from some example above I think, but yeah it can be changed to source.close(done).